integration-tests/src/test/kotlin/com/google/devtools/ksp/test/PlaygroundIT.kt

```diff
@@ -147,19 +146,25 @@ class PlaygroundIT {
         }
     }
 
-    // Disabled for now: ERROR: K2 does not support plugins yet, so please remove -Xuse-k2 flag
-    // Test -Xuse-fir for compilation; KSP still uses FE1.0
-    @Ignore
     @Test
     fun testFirPreview() {
-        val gradleProperties = File(project.root, "gradle.properties")
-        gradleProperties.appendText("\nkotlin.useK2=true")
```

Comment:
Can you confirm what would happen if a user specifies `-Pkotlin.useK2=true` w. and w.o. `languageVersion`? So as to make sure removing `kspTask.compilerOptions.useK2.value(false)` in the above is safe. No need to put it into a test case.

Reply:
This option is deprecated and therefore the errors we saw, it should not be set to true anymore if users want to use FIR.

Reply:
I'd rather spend a few minutes beforehand than hours triage and blaming users' faults later.

Reply:
I meant the behavior when user tries to use this flag in the setup you mentioned in a normal compilation, which is not KSP related. For KSP it doesn't matter, KSP won't get invoked in both scenarios.